documentation/sdk/docs/libraries/rust/how_tos/simple_transaction/simple_transaction/_expected_output.mdx

```diff
@@ -0,0 +1,4 @@
+```plaintext
+Transaction sent: 0x5866ea6342b133e36d6c83c39311c3c65d2c590e7a855b6c10038c68a840a5c5
```

Comment:
eventualyl the goal is to have all examples 100% equal in output, do we still want to do this per language?

Reply:
Yeah I wanted to do that, but somehow forgot it 😅. Let me fix that.
Will follow the Rust example